### DISCUSSION TITLE
Thoughts about the new information banner

### DISCUSSION THREAD

**Discussion by JesusTheHun** [JesusTheHun/storybook-addon-remix-react-router#47] [Polls]
Starting `v2.0.1` a thin information banner is displayed within the add-on panel when a new version of the add-on is available. I would like honest feedback on this.

**Comment by szymonnowak-st** [comment]
I saw this comment saying that there's a new version (2.0.2) available, even though I was on this version already. Other than that, I think it's useful!

**Reply by JesusTheHun** [reply]
Thanks @szymonnowak-st , this is fixed in `2.0.4`.
